Cartfile

```diff
@@ -6,7 +6,7 @@ github "wireapp/FLAnimatedImage" ~> 1.0.17
 github "wireapp/FormatterKit" "1.9.0_Xcode14.3.1"
 github "wireapp/cryptobox-ios" "v1.1.0_xcframework_arm64simulator"
 github "wireapp/libsodium" "1.0.14"
-github "wireapp/PINCache" "2.3-xcode13.1"
+github "wireapp/PINCache" "2.3-xcode14.3.1"
```

Comment:
`wireapp/PINCache` seems to be outdated. Could we sync it with the upstream repo?
Did we add changes to it?

Reply:
yes it's missing one major version, I have added a ticket to deal with the update and the potential breaking changes